### PR TITLE
Update CHANGELOG for v0.92.0

### PR DESCRIPTION
**Pull request by arielshaqed**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,31 @@ Deprecated:
 - Authorization using Role-Based Access Control
 - LDAP & OIDC Embedded Support
```
Comment (ortz) [inline CHANGELOG.md:7]: isn't this covered by the changelog below?

**Review by ortz — APPROVED** [review]
one comment

File: CHANGELOG.md (hunk 1 above)
Comment (arielshaqed) [reply CHANGELOG.md]: OIDC is, LDAP not yet.  Removing "OIDC".

**Review by arielshaqed — COMMENTED** [review]
Nice catch!

**Pull request merged**